ivy/container/general.py

```diff
@@ -368,6 +371,34 @@ def static_gather(
         ret
             New container with the values gathered at the specified indices along
             the specified axis.
+        
+        
+        Examples
+        --------
+        With one :code:`ivy.Container` input:
+
+        >>> x = ivy.Container(a=ivy.array([0., 1., 2.]), \
+                              b=ivy.array([4., 5., 6.]))
+        >>> y = ivy.array([0, 1])
+        >>> print(ivy.static_gather(x, y))
+        >>> print(z)
+        {
+            a: ivy.array(0., 1.),
+            b: ivy.array(4., 5.)
+        }
+
+        With multiple :code:`ivy.Container` inputs:
+    
+        >>> x = ivy.Container(a=ivy.array([0., 1., 2.]), \
+                              b=ivy.array([4., 5., 6.]))
+        >>> y = ivy.Container(a=ivy.array([0, 1]), \
+                              b=ivy.array([1, 2]))
+        >>> y = ivy.Container.static_gather(x, y)
+        >>> print(y)
+        {
+                a: ivy.array([0., 1.]),
+                b: ivy.array(5., 6.)
```

Comment:
missing list brackets around 5.,6.
should be [5., 6.] :)

Reply:
mb I'm stupid and apparently blind, thanks
fixed in f0e50d6959a210d5473ee72705d3204222d0102c